media-video/shotcut/shotcut-19.04.30.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
this is easily ported to EAPI-7, can you do it?

Reply:
no

Reply:
please understand that github is not just a fancy alternative to copy-pasting ebuilds in bugzilla. we expect the submitters to do some work and help them improve at that.